spec/models/user_spec.rb

```diff
@@ -0,0 +1,15 @@
+require "rails_helper"
+
+describe User, type: :model do
+  describe "validations" do
+    it { should validate_presence_of :name}
+    it { should validate_uniqueness_of :name}
```

Comment:
Unless the lesson said otherwise, I believe it should `validate_uniqueness_of :email` instead of name, since people can have the same name, but can't have the same email.